rpcs3/Emu/SysCalls/Modules/cellAvconfExt.cpp

```diff
@@ -1,14 +1,18 @@
 #include "stdafx.h"
+#include "Emu/IdManager.h"
 #include "Emu/Memory/Memory.h"
 #include "Emu/SysCalls/Modules.h"
 
 #include "Ini.h"
+#include "cellAvconfExt.h"
 #include "cellAudioIn.h"
 #include "cellAudioOut.h"
 #include "cellVideoOut.h"
 
 extern Module cellAvconfExt;
 
+u8 gamma;
```

Comment:
Write at least `f32 g_gamma`. And why you even confused float with u8? It doesn't look complicated, why you didn't notice yourself?